src/goto-programs/goto_functions.h

```diff
@@ -16,6 +16,7 @@ Date: June 2003
 
 #include "goto_function.h"
 
+#include "validate_goto_model.h"
```

Comment:
Place below the include from `util` and separate by a blank line.

Reply:
Done.